LumosWeb/api.py

```diff
@@ -38,13 +38,17 @@ def wsgi_app(self, environ, start_response):
 
         return response(environ, start_response)
     
-    def add_route(self, path, handler):
+    def add_route(self, path, handler, allowed_methods=None):
         assert path not in self.routes, "You have already used this route, please choose another route :)"
-        self.routes[path] = handler  # path as an argument.
+        
+        if allowed_methods is None:
+            allowed_methods = ["get", "post", "put", "patch", "delete", "options"]  #  If allowed_methods is not specified by the user, we're allowing all methods to be used.
```

Comment:
The opposite might be possible? If `allowed_methods` is not specified no method can be used?

Reply:
Thank you for your contribution, you are right, it would be more suitable for this framework.